viewer/packages/360-images/src/cache/Image360LoadingCache.ts

```diff
@@ -11,7 +11,7 @@ import { Log } from '@reveal/logger';
 
 export type DownloadRequest = {
   entity: Image360Entity;
-  load360Image: Promise<void>;
+  load360Image: Promise<void | { fullResolutionLoadedPromise: Promise<void> }>;
```

Comment:
I think I'd prefer `undefined | ...` instead of `void | ...` here. Sorry for being a bit back-and-forth on this :sweat: But I promise (:drum:) that this is my final say in the matter